components/camel-micrometer/src/main/docs/micrometer-component.adoc

```diff
@@ -567,17 +575,33 @@ and add a `JmxMeterRegistry` instance:
            HierarchicalNameMapper.DEFAULT));
         return meterRegistry;
     }
-}
 ----
 
-The `HierarchicalNameMapper` strategy determines how meter name and tags are assembled into
-an MBean name.
+CDI::
++
+[source,java]
+----
+    @Produces
+    @Named(MicrometerConstants.METRICS_REGISTRY_NAME))
+    public MeterRegistry getMeterRegistry() {
+        CompositeMeterRegistry meterRegistry = new CompositeMeterRegistry();
+        meterRegistry.add(...);
+        meterRegistry.add(new JmxMeterRegistry(
+           CamelJmxConfig.DEFAULT,
+           Clock.SYSTEM,
+           HierarchicalNameMapper.DEFAULT));
+        return meterRegistry;
+    }
+----
 
+====
+
+The `HierarchicalNameMapper` strategy determines how meter name and tags are assembled into an MBean name.
 
+ifeval::[{show-csb-content} == true]
```

Comment:
Can you remove this ifeval stuff, we do not use that at ASF

Reply:
Yes, I'll remove that. Thanks!

Reply:
done